binderhub/tests/test_build.py

```diff
@@ -16,7 +16,7 @@
 @pytest.mark.parametrize("slug", [
     "gh/binderhub-ci-repos/requirements/d687a7f9e6946ab01ef2baa7bd6d5b73c6e904fd",
     "git/{}/d687a7f9e6946ab01ef2baa7bd6d5b73c6e904fd".format(
-        quote("https://github.com/binderhub-ci-repos/requirements")
+        quote("https://github.com/binderhub-ci-repos/requirements", safe='')
```

Comment:
Why this change to the way the URL is quoted?

Reply:
In the real situation the URL is encoded that way (every special character, including `/`). By default the `quote` function does not url encode `/`, so that's why it needs to specified this way.